### PR TITLE
Tidup and extend all the network maintainers list

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY

* Set maintainers on directory level
* Ensure all `lib/modules/network/` directories are listed
* Add `module_utils`
* Add `plugin/action`
* Add `plugin/connection`
* Add `plugins/terminal`

##### ISSUE TYPE

 - Bugfix Pull Request

### CODE REVIEW

**Comment by gundalow** [comment]
@jctanner as requested this adds in items such as
`$module_utils/dellos: skg-net` which should match `$module_utils/dellos6.py` `$module_utils/dellos10.py`

Is that format OK for you, if so I'll raise an issue against the bot

**Review by jctanner — APPROVED** [review]
shipit

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -875,6 +835,93 @@ files:
     - core inventory
     - inventory
     - inventory parsing
+  $module_utils/a10.py: ericchou1 mischapeters
+    labels:
+    - networking
```
Comment (bcoca) [inline .github/BOTMETA.yml]: this is invalid YAML, once you expand the entry into a dict, you cannot have 'inline maintainers', they need to go into their own key.

**Review by bcoca — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
https://github.com/ansible/ansibullbot/issues/688 tracks updating the bot

**Pull request merged**